tests/compilers/test_ast.py

```diff
@@ -404,6 +404,12 @@ def test_ast_tuple():
     assert type(code) == ast.Tuple
 
 
+def test_argument_destructuring():
+    """ Ensure argument destructuring works. """
+    can_compile("(fn [[a b]] (print a b))")
```

Comment:
> Would it go in native_tests/language.hy?

Sure? (said with the intonation of a question, as if to allude to [your experience with](https://github.com/hylang/hy/commits?author=kirbyfan64) how things are done around here outweighing [mine](https://github.com/hylang/hy/commits?author=zackmdavis))